--- libmimedir-0.2.1/mimedir/mimedir-vcard.h~	Tue Jan 14 00:04:20 2003
+++ libmimedir-0.2.1/mimedir/mimedir-vcard.h	Sat Sep 20 13:34:48 2003
@@ -37,12 +37,11 @@
 #define MIMEDIR_VCARD_LONGITUDE_MAX	(+180.0)
 #define MIMEDIR_VCARD_LONGITUDE_MIN	(-180.0)
 
-enum _MIMEDirVCardKeyType {
+typedef enum {
 	MIMEDIR_VCARD_KEY_UNKNOWN,	/* Key type is not known */
 	MIMEDIR_VCARD_KEY_X509,		/* Key is a X.509 key */
 	MIMEDIR_VCARD_KEY_PGP		/* Key is a PGP/GnuPG key */
-};
-typedef enum _MIMEDirVCardKeyType MIMEDirVCardKeyType;
+} MIMEDirVCardKeyType;
 
 struct _MIMEDirVCardKey {
 	gchar *key;
